docs/stcal/ramp_fitting/description.rst

```diff
@@ -284,3 +284,18 @@ For the optional output product, the variance of the slope due to the Poisson
 noise of the segment-specific slope is written to the VAR_POISSON extension.
 Similarly, the variance of the slope due to the read noise of the
 segment-specific slope  is written to the VAR_RNOISE extension.
+
+Data Quality Propagation
+=================
+
+If all groups in an integration are flagged as DO_NOT_USE or SATURATED, then
+the rateints product will be flagged as DO_NOT_USE.  For the rates product, the
```

Comment:
Not the entire rateints product. The plane of the DQ array corresponding to that integration will be flagged.  Also, "the rates product" should be "the *rate* product".